functions/src/api/v1/rooms/editRoom.ts

```diff
@@ -72,5 +84,26 @@ export const editRoom = wrap(async (req: Request, res: Response) => {
 
     await RoomDao.update(dataToEdit)
 
+    if (req.body.sendsAt) {
+        await updateReminders(roomId, req.body.sendsAt)
+    }
+
     res.send()
 })
+
+const updateReminders = async (roomId: RoomId, sendsAt: string) => {
+    const sendsAtValues = parseSendsAt(sendsAt)
+    if (sendsAtValues.length > 0) {
+        const oldReminders = await ReminderDao.listByRoomId(roomId)
+
+        for (const sendAt of sendsAtValues) {
+            const sendAtMillis = parseInt(sendAt) * 1000
+            const sendAtTimestamp = Timestamp.fromMillis(sendAtMillis)
+            await ReminderDao.add(roomId, sendAtTimestamp)
+        }
+
+        for (const reminder of oldReminders) {
```

Comment:
this is shouldn't be executed before add?

Reply:
the goal is to prevent removing any reminders if the add op fail, so it's kind like a transaction but ery very basic.